chromadb/server/fastapi/__init__.py

```diff
@@ -179,28 +181,28 @@ def add(self, collection_name: str, add: AddEmbedding):
             raise HTTPException(status_code=500, detail=str(e))
         return result
 
-    def update(self, collection_name: str, add: UpdateEmbedding):
+    def update(self, collection_id: str, add: UpdateEmbedding):
         return self._api._update(
             ids=add.ids,
-            collection_name=collection_name,
+            collection_id=UUID(collection_id),
             embeddings=add.embeddings,
             documents=add.documents,
             metadatas=add.metadatas,
         )
 
-    def upsert(self, collection_name: str, upsert: AddEmbedding):        
+    def upsert(self, collection_id: str, upsert: AddEmbedding):
         return self._api._upsert(
-            collection_name=collection_name,
+            collection_id=UUID(collection_id),
```

Comment:
UUID can throw an error if collection_id is malformed, do we want to handle this in a more legible way?